packages/terra-clinical-result/src/terra-dev-site/doc/ClinicalResult/FlowsheetResultCell.3.doc.mdx

```diff
@@ -44,6 +44,7 @@ The flowsheet cell for clinical results is constructed by providing the same str
 
 Currently the Flowsheet expects an array of one or more results - and if there are multiple, it will display the first in the array, with an indication of additional results. Updates are in progress to also look at the performedDateTime value if provided to evaluate which result is the most recent to appropriately display, regardless of which order index position in the array it is.
 
+Note: The Flowsheet Result Cell has a minimum width requirement of 10em (10 times the font-size provided to it) to prevent a blood pressure value from being truncated. For normal Terra theme usage, this will typically be 140px (assuming for base font-size at 14px). 
```

Comment:
```suggestion
Note: The Flowsheet Result Cell has a minimum width requirement of 10em (10 times the font-size provided to it) to prevent a blood pressure value from being truncated, provided the unit of measure is hidden and not displayed. For normal Terra theme usage, this will typically be 140px (assuming for base font-size at 14px). 
```

Reply:
Fixed here https://github.com/cerner/terra-clinical/pull/637/commits/788a6eed719217b512d85dbc8c1fa1c37d747e50#diff-074b0ccea712cf9722ad97484fd56bd1R47